### DISCUSSION TITLE
Mark/Hide certain endpoints

### DISCUSSION THREAD

**Discussion by ciaransweet** [rohit-gohri/redocusaurus#319] [Q&A]
Hey folks,

Quick one I hope - Is there a way we can mark some paths as `internal` or similar?

Let's say we have a spec and it contains

```
/healthcheck # Probably shouldn't be documented externally on docs
/acollection # Should be documented externally on docs
```

We don't want to have two separate specs because in theory, those paths do both exist, but for people who consume our API, they don't need to see `/healthcheck` in our docs...

Thanks for any suggestions!

**Comment by rohit-gohri** [comment]
Hey

Thanks for using the project but this would be out of scope since the rendering is done by redoc and this project is an integration with it.

The best way to do this IMO is to use some custom property and filter it out from your public documentation in the yaml before building
